webapp/views.py

```diff
@@ -679,19 +679,22 @@ def marketo_submit():
         ],
     }
 
-    # Enrichment data for global enrichment form (id:4198)
-    enrichment_fields = {"email": form_fields["email"]}
-    try:
-        ip_location = ip_reader.get(client_ip)
-        if ip_location and "country" in ip_location:
-            enrichment_fields["country"] = ip_location["country"]["iso_code"]
-    except Exception:
-        pass
-
-    enriched_payload = {
-        "formId": "4198",
-        "input": [{"leadFormFields": enrichment_fields}],
-    }
+    if "email" in form_fields:
```

Comment:
We just want this condition around line 684 as we would expect line 736 to still submit all passed fields.

Reply:
Ok. In that case I initiated the variable with `None` otherwise we'll get variable not defined if it fails the check